libs/watt/package/text-field/watt-text-field.component.ts

```diff
@@ -47,6 +47,9 @@ export type WattInputTypes = 'text' | 'password' | 'email' | 'number' | 'tel' |
   selector: 'watt-text-field',
   styleUrls: ['./watt-text-field.component.scss'],
   encapsulation: ViewEncapsulation.None,
+  host: {
+    '[attr.watt-field-disabled]': 'isDisabled',
```

Comment:
suggestion: needs be called


```suggestion
    '[attr.watt-field-disabled]': 'isDisabled()',
```